#!/usr/bin/env python
PACKAGE = "viconros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Log_to_file_once", bool_t, 0, "Log_to_file_once", False);
gen.add("Enable_log_to_file", bool_t, 0, "Enable_log_to_file", False);
gen.add("Correct_start", bool_t, 0, "Enable_Correcting", False);

gen.add("EV_POS_X", double_t, 0, "Deviation_X_for_Forward(m)", 0.0, -1.0, 1.0);
gen.add("EV_POS_Y", double_t, 0, "Deviation_Y_for_Right(m)", 0.0, -1.0, 1.0);
gen.add("EV_POS_Z", double_t, 0, "Deviation_Z_for_Down(m)", 0.0, -1.0, 1.0);

gen.add("BIAS_POS_X", double_t, 0, "BIAS_X_for_world(m)", 0.39515, -1.0, 1.0);
gen.add("BIAS_POS_Y", double_t, 0, "BIAS_Y_for_world(m)", 0.05305, -1.0, 1.0);
gen.add("BIAS_POS_Z", double_t, 0, "BIAS_Z_for_world(m)", -0.6985, -1.0, 1.0)

exit(gen.generate(PACKAGE, "viconros", "uam_init_"))
